src/components/curator-actions/fetch-signatories.ts

```diff
@@ -49,3 +80,53 @@ export async function fetchMultisigSignatories(curatorAddress: string): Promise<
 		return [];
 	}
 }
+
+/**
+ * Fetches multisig accounts that the signatory is part of.
+ *
+ * @param signatory signatory address.
+ * @returns array of multisig accounts that the signatory is part of, an empty array in case of error.
+ */
+export async function fetchMultisigAccount(signatory: string): Promise<MultisigInfo[]> {
```

Comment:
```suggestion
export async function fetchMultisigInfo(signatory: string): Promise<MultisigInfo[]> {
```
Would it make sense to align the name of the function and its return type?

Reply:
[732a285](https://github.com/galaniprojects/Polkadot-Bounty-Manager/pull/491/commits/732a2850f3ac8217422091fdeaef382b990d8aa8)